plugins/source/facebookmarketing/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	golangci-lint run --config ../../.golangci.yml
+
+.PHONY: gen-docs
+gen-docs:
+	go run main.go doc --format json docs
+	rm docs/__tables.json
+
+	rm -rf ../../../website/tables/facebookmarketing
```

Comment:
we use this code a lot.
@erezrokah maybe we can move it to root of source plugins and call with plugin dir param?
like `../../gen-docs facebookmarketing`
and contents like
```bash
#!/usr/bin/env bash
BASEDIR=$(dirname "$0")

rm -rf "${BASEDIR}/../../website/tables/${1}"

go run "${BASEDIR}/${1}/main.go" doc "${BASEDIR}/../../website/tables/${1}"
sed -i.bak -e 's_(\(.*\).md)_(tables/\1)_' "${BASEDIR}/../../website/tables/${1}/README.md"
mv "${BASEDIR}/../../website/tables/${1}/README.md" "${BASEDIR}/../../website/tables/${1}/README.md" "${BASEDIR}/../../website/pages/docs/plugins/sources/${1}/tables.md"
sed -i.bak -e 's_(\(.*\).md)_(\1)_' "${BASEDIR}/../../website/tables/${1}/*.md"
rm -rf "${BASEDIR}/../../website/tables/${1}/*.bak"
```

Reply:
What do you mean by move to root? I think we'd still want to have the code duplicated, otherwise if we put the script under `plugins/source` for example we'll need to run the CI for every plugin when `plugins/source` changes

Reply:
Root = `plugins/source` dir.
We might want to run those, yes.
But, maybe not required initially, as every change to this file should also include manual generation of all source plugins?